src/ast.rs

```diff
@@ -79,8 +76,8 @@ fn get_ts_config(tsx: bool, dts: bool) -> TsConfig {
   }
 }
 
-impl<'a> From<&'a MediaType> for Syntax {
-  fn from(media_type: &'a MediaType) -> Self {
+impl<'a> From<MediaType> for Syntax {
+  fn from(media_type: MediaType) -> Self {
```

Comment:
Since the `MediaType` is already `Copy`, I think it's a bit nicer to just pass it around instead of using references.

Reply:
But doesn't copy do unnecessary heap allocations?  The rest of everything we pass around is references already.

Reply:
From my understanding this won't cause any heap allocations because the enum is represented as i32 (because of `repr(i32)` in this case, usually it would be `u8` I think for an enum with this many members). I often see see these "value" enum types passed around like a value (like an `i32` instead of `&i32`) than needing to be a reference. I don't have strong opinions though and can revert it?

https://stackoverflow.com/a/44695401/188246

Reply:
Naw, it is fine.  Thanks for the explanation.